starter-code/src/array-functions.js

```diff
@@ -1,33 +1,39 @@
-var ArrayFunctions = function (){}
+const ArrayFunctions = function (){}
 
 // 1) Define a function that takes an array with numbers and prints all the elements of the array, separated by " --- "
 // ArrayFunctions.printSpecial([12, 33, 144, 122])
 // 12 -- 33 -- 144 -- 122
 
-ArrayFunctions.printSpecial = function (array) {
-  return array.join(" --- ");
-};
+ArrayFunctions.printSpecial = array => array.join(" --- ");
```

Comment:
you should've used static methods, here you have them well explained:
http://odetocode.com/blogs/scott/archive/2015/02/02/static-members-in-es6.aspx